cli/src/movement_util.rs

```diff
@@ -25,16 +25,24 @@ use crate::cli_util::{short_commit_hash, WorkspaceCommandHelper};
 use crate::command_error::{user_error, CommandError};
 use crate::ui::Ui;
 
-pub enum Direction {
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub(crate) struct MovementArgs {
```

Comment:
nit: I wouldn't add `Copy` here as we might add more parameters to this struct.

Reply:
Makes sense. Done. Thanks.